velox/docs/monthly-updates/august-2023.rst

```diff
@@ -0,0 +1,65 @@
+****************
+August 2023 Update
+****************
+
+Documentation
+=============
+
+* Add :doc:`documentation on adaptive disabling of partial agregation </develop/aggregations>`.
+* Add :doc:`documentation on Aggregate::toIntermediate() API </develop/aggregate-functions>`.
+* Add :doc:`documentation on connectors </develop/connectors>`.
+
+Core Library
+============
+
+* Add support for remote functions. :pr:`6253`
+* Add support for accessing sub-fields by index.
+* Add support for DECIMAL in UnsafeRowSerializer.
+* Add checks for verifying internal state of a vector.
+* Enforce coalesce and switch expression to ensure that all inputs are of same type.
+* Fixes for handling exceptions in try_cast from JSON.
+
+
+Presto Functions
+================
+
+* Add :func:`split_to_map` , :func:`array_remove`, :func:`laplace_cdf`, :func:`date_format`, :func: `format_datetime` functions.
+* Add :func:`gamma_cdf`, :func:`poisson_cdf` , `last_day_of_month` and 2 argument :func:`trim` functions.
+* Add :func:`wilson_interval_lower` , :func:`wilson_interval_upper` functions.
+* Optimize :func:`array_constructor` function.
+
+
+Spark Functions
+===============
+
+* Add :spark:func:`date_add`, :spark:func:`date_sub`, :spark:func:`pmod` , :spark:func:`week_of_year`
+* Add :spark:func:`max_by` , :spark:func:`min_by` spark aggregate functions.
+* Add :spark:func:`dayofyear` , :spark:func:`dayofmonth`, :spark:func:`dayofweek` functions.
+* Add :spark:func:`row_number` window function.
+
+
+Hive Connector
+==============
+
+* Add support for writing to `Google Cloud Storage <https://cloud.google.com/storage>`_. :pr:`5685`
+* Add support for writing data to HDFS via INSERT INTO/CTAS.
```

Comment:
Drop 'data' from 'writing data' to match the first bullet point.

Spell out CTAS: CREATE TABLE AS SELECT (CTAS).

Was CTAS broken specifically for HDFS? Who can provide more context about this change? CC: @majetideepak

Reply:
CTAS was broken for HDFS. It did not have a FileSink and the files were being written to the local FS instead of HDFS.
An `HdfsFileSink` was added.

Reply:
Thank you for clarifying, Deepak. Maybe, this can be worded as 

`Fix CREATE TABLE AS SELECT (CTAS) queries that are writing to HDFS. :pr:NNN`

Reply:
or, `Fix writing  to HDFS as part of CREATE TABLE AS SELECT (CTAS) query.`